e2e/home/index.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { siteExternalLinks } from '@/app/config/site';
+import { test, expect, Page } from '@playwright/test';
+
+const EXPECTED_TITLE = 'Biblioteca de Zines';
+const LINK_TEXT_EXPLORE = 'Explorar :)';
+const LINK_TEXT_NEWSLETTER = 'Newsletter';
+const LINK_TEXT_SUBMIT = 'Sugerir Zine';
+
+test.describe('Homepage', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/');
+  });
+
+  test('has correct title', async ({ page }) => {
+    await expect(page).toHaveTitle(EXPECTED_TITLE);
+  });
+
+  test('has correct content', async ({ page }) => {
+    await expect(page.locator('h1')).toHaveText(EXPECTED_TITLE);
+    await expect(page.getByRole('link', { name: LINK_TEXT_EXPLORE })).toBeVisible();
+    await expect(page.getByRole('link', { name: LINK_TEXT_NEWSLETTER })).toBeVisible();
+    await expect(page.getByRole('link', { name: LINK_TEXT_SUBMIT })).toBeVisible();
+  });
+
+  test('explore button navigates to the zines page', async ({ page }) => {
```

Comment:
eu _particularmente_ prefiro a sintaxe `it` ao invés de `test` pq eu acho mais "fácil de entender".

Vc digita: test explore button navigates to the zine pages
vs
it has buttons that navigates to the zine pages

saca?

aí ficaria assim:
```suggestion
  it('has a explore button that navigates to the zines page', async ({ page }) => {
```

mas é 200% opcional, é só uma preferência pessoal mesmo :D

Reply:
hmmmm nem tinha me tocado que existe essa sintaxe `it`, achei que fica mais fácil mesmo. vou aplicar e volto aqui!